browser/resources/settings/brave_add_site_dialog/brave_add_site_dialog_behavior.ts

```diff
@@ -0,0 +1,23 @@
+// @ts-nocheck
```

Comment:
let's not add new files with `@ts-nocheck`. Please add types.

Reply:
done, thanks!